project-koku/koku
# [COST 1167] : Fix null in source_uuid & make sure all source_uuids show up for others

#### To test 
On master 
1. populate the db
```
make docker-reinitdb-with-sources
make load-test-customer-data
```
2. Hit this endpoint http://127.0.0.1:8000/api/cost-management/v1/reports/openshift/costs/?filter[time_scope_units]=month&filter[time_scope_value]=-1&filter[resolution]=monthly&group_by[cluster]=*&filter[limit]=1

Notice the others only has one source uuid & there is a null value for the cluster that shows 
**Repeat on this branch to see that the null value is removed and the appropriate amount of source_uuids are in the others data**

#### Smoke testes results 
```
================================== 11 failed, 5938 passed, 13 skipped, 705 deselected, 1 error in 3321.43s (0:55:21) ==================================
```
